lib/linguist/languages.yml

```diff
@@ -8468,10 +8468,28 @@ crontab:
   language_id: 705203557
 desktop:
   type: data
+  # https://en.wikipedia.org/wiki/INI_file
```

Comment:
These are all in the wrong place. If they're INI files, we already have a [section](https://github.com/github-linguist/linguist/blob/1a61b82c42754ff84f1a3e108150a2a7461c10bc/lib/linguist/languages.yml#L3056) for that.

Reply:
> These are all in the wrong place. If they're INI files, we already have a [section](https://github.com/github-linguist/linguist/blob/1a61b82c42754ff84f1a3e108150a2a7461c10bc/lib/linguist/languages.yml#L3056) for that.

But then the already existing (systemd) ".service" is in the wrong place as well? Do you want me to move that one too? Otherwise it's inconsistent. Full disclaimer: I don't yet fully understand what difference "desktop" (which are also INI) VS "ini" language makes either way. What I could do is raise a separate small PR, just to first move . service?

Reply:
🤔 good point. Leave these here. I'd forgotten @Alhadis added `.service` in https://github.com/github-linguist/linguist/pull/5768 and makes the point the files are based on desktop which is essentially INI.

We def don't need the comments and only really want files and extensions which meet our usage requirements, along with a sample of each.

Reply:
@lildude Yeah, something I didn't address in #5768 was the observation that the "Desktop" language ought to be rolled into INI (which appears to be the correct course of action, AFAICT), because apart from having distinct syntax highlighting grammars, each entry refers to essentially the same format.

I only added `.service` to Desktop's file extensions because its usage appeared to be affiliated with <samp>systemd(1)</samp> and I didn't have time to look into whether or not it was worth merging Desktop into INI.